src/seedu/addressbook/AddressBook.java

```diff
@@ -208,7 +208,18 @@
 
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        if (args.length >= 2) {
```

Comment:
This commit is unrelated to T2A2.
Tip: Avoid accumulating commits in your pull request by branching properly!